play/benchmarker/project/activator-sbt-eclipse-shim.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "2.3.0")
```

Comment:
What do files like these do?  For example, does this one do something related to the actual application, or does it add something to the developer's IDE?